src/uwtools/resources/jsonschema/fv3.jsonschema

```diff
@@ -142,6 +155,7 @@
         }
       },
       "required": [
+        "diag_table",
```

Comment:
This is required for the successful completion of fv3, so I strengthened the schema.